gdal/src/test/scala/geotrellis/raster/gdal/GDALRasterSourceSpec.scala

```diff
@@ -110,5 +113,50 @@ class GDALRasterSourceSpec extends FunSpec with RasterMatchers with GivenWhenThe
       lazy val tsource = GeoTiffRasterSource(uri)
       source.metadata.attributes.mapValues(_.toUpperCase) shouldBe tsource.metadata.attributes.mapValues(_.toUpperCase)
     }
+
+    describe("should derive the cellType consistently with GeoTiffRasterSource") {
```

Comment:
This test passes locally with GDALWarp bindings [1.0.1-fix_metadata_calls-SNAPSHOT](https://oss.sonatype.org/content/repositories/snapshots/com/azavea/geotrellis/gdal-warp-bindings/1.0.1-fix_metadata_calls-SNAPSHOT)